.changeset/small-bats-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@fake-scope/fake-pkg': minor
```

Comment:
```suggestion
'@guardian/libs': minor
```

Reply:
Sorry, I charged ahead with the automated changeset without looking closely enough 🙈